docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -54,14 +63,48 @@ Use the following code snippet to perform generation with OpenVINO GenAI API:
          #include <iostream>
 
          int main(int argc, char* argv[]) {
-            std::string model_path = argv[1];
+            std::string model_path = "TinyLlama";
             ov::genai::LLMPipeline pipe(model_path, "NPU");
             std::cout << pipe.generate("The Sun is yellow because", ov::genai::max_new_tokens(100));
          }
 
 Additional configuration options
 ################################
 
+Prompt and response length options
+++++++++++++++++++++++++++++++++++
+
+The LLM pipeline for NPUs leverages the static shape approach, optimizing execution performance, while potentially introducing certain usage limitations. By default, the LLM pipeline supports input prompts up to 1024 tokens in length. It also ensures that the generated response contains at least 150 tokens, unless the generation encounters the end-of-sequence (EOS) token or the user explicitly sets a lower length limit for the response.
+
+You may configure both the 'maximum input prompt length' and 'minimum response length' using the following parameters:
+
+- ``MAX_PROMPT_LEN``: Defines the maximum number of tokens that the LLM pipeline can process for the input prompt (default: 1024).
+- ``MIN_RESPONSE_LEN``: Specifies the minimum number of tokens that the LLM pipeline will generate in its response (default: 150).
+
+Use the following code snippet to change the default settings:
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+      .. code-block:: python
+
+         pipeline_config = { "MAX_PROMPT_LEN": 1500, "MIN_RESPONSE_LEN": 500 }
+         pipe = ov_genai.LLMPipeline(model_path, "NPU", pipeline_config)
+
+   .. tab-item:: C++
+      :sync: cpp
+
+      .. code-block:: cpp
+
+         ov::AnyMap pipeline_config = { { "MAX_PROMPT_LEN",  1500 }, { "MIN_RESPONSE_LEN", 500 } };
+         ov::genai::LLMPipeline pipe(model_path, "NPU", pipeline_config);
+
+
+Compilation options
++++++++++++++++++++
+
 Compiling models for NPU may take a while. By default, the LLMPipeline for the NPU
```

Comment:
There was also concern if we should still recommend such options (CC: @dmatveev)

Reply:
Removed